src/patternfly/components/Alert/alert.scss

```diff
@@ -236,8 +236,6 @@
   padding-block-start: var(--#{$alert}__action-group--PaddingBlockStart);
 
   > .#{$button} {
-    --#{$button}--m-link--m-inline--hover--TextDecoration: none;
-
     &:not(:last-child) {
       margin-inline-end: var(--#{$alert}__action-group__c-button--not-last-child--MarginInlineEnd);
     }
```

Comment:
Nit but if this is just intended to hold actions, it would be a lot better as a flex layout (or using the action list component)
* it would make the spacer we're using here as a margin (`--pf-t--global--spacer--gap--action-to-action--default`) the actual space between actions, instead of what it is now, which is the spacer + a space character
    <img width="371" alt="Screenshot 2024-07-01 at 4 13 45 PM" src="https://github.com/patternfly/patternfly/assets/35148959/d46e772a-eb58-4ce9-bfc6-c22e3b8131eb">
* we could add a row gap
* it will let the actions touch the right edge of the container if they wrap - they don't right now because there will always be a margin to the right of an action
* We can remove all button styling from this block